### ISSUE TITLE
Support multiomics data sets

### ISSUE THREAD

**Issue by kevinrue** [iSEE/iSEE#659]
Considering the number of times that was brought up during conference presentations and workshops, I'm surprised not finding an issue requesting this feature yet.

Following recent discussions, there are two main avenues to approach this:

- Support `MultiAssayExperiment` objects
- Support the`altExps` slot of `SingleCellExperiment` objects.

The former likely requires more work to access the various SE objects nested in the MAE.
Notably, MAE is not an SE extension itself, which doesn't immediately fit in the iSEE paradigm.

The latter may be more immediately applicable, since SCE is already supported by `iSEE()`.
This approach is more about adding UI elements to switch between active and alternative assays.

====

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by RiboRings** [comment]
Hi! This feature has indeed great potential. I thought many times about having a button that lets you switch from the main experiment to altExps and vice versa. This would be particularly useful for microbiome data in TreeSE containers, because you often store experiments at different taxonomic levels in the altExp slots.

A few practical questions would be where to put this altExp choice and what observable is best considering there might be multiple altExps? And also what happens to the panels when changing to another experiment, is their status preserved or are changes lost?

Looking forward to this feature!

**Comment by kevinrue** [comment]
Yup. All those questions and more are reasons why this feature is still on hold.
Also because I'm still trying to get funding to work on iSEE "officially" and not just on my spare time :/

Thanks for the support!
